.github/workflows/nomad.yml

```diff
@@ -7,7 +7,6 @@ on:
   pull_request:
     paths:
       - 'deployments/nomad/**'
-      - '.github/workflows/nomad.yml'
```

Comment:
@crobert-1 Let's keep this so that the tests run if the workflow file is changed.

Reply:
Whoops, went too quickly through rebase conflicts, thanks for catching!